internal/core/src/index/VectorDiskIndex.cpp

```diff
@@ -510,6 +510,14 @@ VectorDiskAnnIndex<T>::update_load_json(const Config& config) {
     return load_config;
 }
 
+template <typename T>
+ResourceUsage
+VectorDiskAnnIndex<T>::GetResourceUsage() const {
+    // TODO: we can't get the memory usage of vector index now.
+    auto disk_size = file_manager_->GetLocalFileSize();
+    return ResourceUsage{0, disk_size};
```

Comment:
this need to double check with @liliu-z 
i believe there is a way to estimate memory utilizations